scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -28,21 +28,11 @@ def call(final pipelineContext) {
   // Job will execute PR_STAGES only if these are green.
   def SMOKE_STAGES = [
     [
-      stageName: 'Py2.7 Smoke', target: 'test-py-smoke', pythonVersion: '2.7',timeoutValue: 8,
```

Comment:
this does not look right, we want to remove all smokes?

Reply:
No, but if I want to test stuff quickly, I want this to run in smoke tests. Solely. Told you above to wait for a while until I finish trying out some stuff. Sorry for the inconvenience.

Reply:
Reverted. Not it's ok.